src/utils/DarCollectionUtils.js

```diff
@@ -267,10 +267,16 @@ export const getMatchDataForBuckets = async (buckets) => {
         ).referenceId;
         return isNil(dataAccessReferenceId);
       });
+
       if(!isNil(dataAccessReferenceId)) {
         idsArr.push(dataAccessReferenceId);
         electionIdBucketMap[dataAccessReferenceId] = bucket;
       }
+
+      dars.forEach((dar) => {
```

Comment:
Ah, we could do that too, I suppose - I was following the language on the ticket that said it should be searching by darId as well. I assumed there would be some data out there that would have data for the darId because of that, but that assumption may be wrong! Just to be sure, would it be possible to ever get data returned when requesting by the reference id of the dar?